CONTRIBUTING.rst

```diff
@@ -167,3 +167,30 @@ To speed up the integration process, it helps if on GitHub you `allow
 maintainers to edit your pull request
 <https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/>`_
 so they can fix small issues autonomously.
+
+.. rubric: Review guidelines
+
+A pull request will probably be reviewed by someone in a different location,
+maybe even a different time zone. Therefore, avoiding blockings and
+misunderstandings is key. The following ideas could help ease and speed
+up the review process:
+
+- Be clear when something is ready to reviewed (see `WIP` above).
+- If a PR requires changes, say so explicitly, but politely (use emojis!).
+- When something is ready to be merge say it explicitly. If the reviewer does
+not have permission to merge, it should ping the corresponding person.
+- When reviewing, rate your comments. This would help the other person know If
+a comment must be addressed (e.g. it has substantial implications) or it is
+something the reviewer thought of but is a matter of "take it or leave it". A
+proposed scale is:
+  1. Comment/Shelved: Something worth mentioning, a doubt or question (comment).
+  If it will be treated later on as part of a different issue it is "shelved".
+  In case of shelving reference the issue.
+  2. Minor: Something that the reviewer thinks might need change. However, it
+  is not blocking, it is up to the developer to choose if and how to change
+  it. It can be merged!
+  3. Moderate: Something that requires further discussion (e.g. breaking
+  changes). It cannot be merged, unless explicitly said (e.g. choose a
+  solution proposed by the reviewer and implement it).
```

Comment:
Moderate: I am not sure this point is clear. It looks like that the reviewer proposed several different solutions, the creator of the PR then chooses one and then merge? After having implemented the chosen solution, I guess another review is required. Maybe I did not understand correctly.

Reply:
The "choose a solution" is an example. Commonly, "moderate" means don't merge right away, engage with the review comment and see where that leads: sometimes a change is needed, sometimes the reviewer missed something and no changes is needed. In any case, a back and forth will occur, but the back and forth is usually short. For example:

- reviewer: I thought we wanted to require this field?
- submitter: `<checks up on the question>` Ah you're right! Let me fix that. <pushes changes> `<alerts reviewer>`
- reviewer: `<takes another look>` Thanks, all good! `<approves if no other outstanding issues>`

Reply:
Exactly. Maybe this is a "loose end" we might have done sometimes. There are changes that are trivial (e.g. add `requires=True` to a marshmallow schema) but it has an impact so the comment is "moderate". I guess in those cases what we have done is:
1- Comment the `moderate` change
2- State that it can be merged
3- PR creator does the change (trivial) and can merge

Ofc, in this case we are assuming PR creator/reviewer are both maintainers... Maybe we shouldnt?